webpack/scenes/Subscriptions/__tests__/SubscriptionsActions.test.js

```diff
@@ -18,21 +18,35 @@ import {
   loadTableColumnsSuccessAction,
 } from './subscriptions.fixtures';
 import {
+  handleTask,
+  pollTasks,
+  cancelPollTasks,
+  resetTasks,
   loadSubscriptions,
   updateQuantity,
   loadAvailableQuantities,
   loadTableColumns,
   updateSearchQuery,
   openDeleteModal,
   closeDeleteModal,
-  openTaskModal,
-  closeTaskModal,
   disableDeleteButton,
   enableDeleteButton,
 } from '../SubscriptionActions';
 
+import { getTaskPendingResponse, getTaskSuccessResponse } from '../../Tasks/__tests__/task.fixtures';
+
 const mockStore = configureMockStore([thunk]);
-const store = mockStore({ subscriptions: Immutable({}) });
+const store = mockStore(Immutable({
+  intervals: {
+    SUBSCRIPTIONS_TASK_SEARCH: 5,
```

Comment:
Can you use [jest.useFakeTimers()](https://jestjs.io/docs/en/timer-mocks) instead of using a real 5ms interval?

Reply:
I'm unclear on what that would accomplish since the test isn't actually executing the interval polling but making a determination based on what's in the store. Maybe I'm missing your point.

Reply:
I see, I guess it's not applicable here.  Carry on  :yum: